conf/base.config

```diff
@@ -15,7 +15,6 @@ process {
     memory = { 6.GB   * task.attempt }
     time   = { 8.h    * task.attempt }
 
-    // memory errors which should be retried. otherwise error out
     errorStrategy = { task.exitStatus in ((130..145) + 104 + 175) ? 'retry' : 'finish' }
```

Comment:
```suggestion
    // memory errors which should be retried. otherwise error out
    errorStrategy = { task.exitStatus in ((130..145) + 104 + 175) ? 'retry' : 'finish' }
```